vochain/offchaindatahandler/offchaindatahandler.go

```diff
@@ -87,7 +92,7 @@ func (d *OffChainDataHandler) Commit(_ uint32) error {
 			go d.enqueueOffchainCensus(item.censusRoot, item.uri)
 		case itemTypeElectionMetadata, itemTypeAccountMetadata:
 			log.Infow("importing data", "type", itemTypesToString[item.itemType], "uri", item.uri)
-			go d.enqueueMetadata(item.uri)
+			go d.enqueueMetadata(item)
```

Comment:
same here, i pass the whole item so i can have access to all the info about the item queued